presto-hive/src/main/java/com/facebook/presto/hive/HiveSplitManager.java

```diff
@@ -69,6 +69,7 @@
         implements ConnectorSplitManager
 {
     public static final String PRESTO_OFFLINE = "presto_offline";
+    private static final String STRUCT_PREFIX = "struct<";
```

Comment:
Is it possible to use "RowType" instead of "struct"? Or may be use `com.facebook.presto.hive.HiveType.java`.